examples/basics/basics.ipynb

```diff
@@ -1,441 +1,468 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.    print("Number of datasets in your org:", len(list(datasets)))
</pre><p>I don't think we want users pulling all projects and datasets. I would just do something like:</p><pre class="ql-syntax" spellcheck="false">dataset = next(client.get_dataset())
print("Dataset ID: ", dataset.uid)
print("Dataset Name:" , dataset.name)
</pre><p>Maybe just explain that both get_datasets() and get_projects() return a PaginatedCollection object which is just an iterable object. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/745/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Sounds good.</p>